pkg/placement/fake/ocm.go

```diff
@@ -0,0 +1,67 @@
+//go:build unit
+
+package fake
+
+import (
+	"context"
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"sigs.k8s.io/gateway-api/apis/v1beta1"
+
+	"github.com/Kuadrant/multicluster-gateway-controller/pkg/dns"
+	testutil "github.com/Kuadrant/multicluster-gateway-controller/test/util"
+)
+
+type FakeGatewayPlacer struct{}
```

Comment:
Is this different than https://github.com/Kuadrant/multicluster-gateway-controller/blob/e9870c7a822920d24df19fdcf9bb941ebbb165f8/test/integration/helper_test.go#L41 ?

Reply:
Yes, there are 3 implementations of it.  I'm looking at it now, but it will be #330?

Reply:
Yes, it is. And #330 would addres it

Reply:
Ack